it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/RegistrationControllerISpec.scala

```diff
@@ -193,6 +199,9 @@ class RegistrationControllerISpec extends ComponentSpecHelper with AuthStub with
           stubRetrieveBusinessVerificationStatus(testJourneyId)(status = OK, body = Json.toJson(BusinessVerificationPass))
           stubRegisteredSocietyRegister(testCompanyNumber, testCtutr)(status = OK, body = testFailedRegistration)
           stubStoreRegistrationStatus(testJourneyId, testFailedRegistration)(status = OK)
+          stubRetrieveIdentifiersMatch(testJourneyId)(status = OK, body = true)
+          stubRetrieveRegistrationStatus(testJourneyId)(status = OK, body = Json.toJson(testFailedRegistration))
+
```

Comment:
Is the auditing being verified here?

Reply:
I've fixed this with a new method that verifies the explicit audit sent